modules/volunteers/vue/hourTable.js

```diff
@@ -148,7 +148,12 @@ export default {
         if (column.value == 'total_hours' || column.value == 'hours') {
           return this.printHours(record[column.value]);
         } else {
-          return record[column.value];
+          if (column.type == 'Date') {
+            var d = new Date(Date.parse(record[column.value]));
```

Comment:
I think we should be able to leverage `new Date(record[column.value]);` here, assuming the value is a date string in ISO-8601 format. No need to `Date.parse()` at all, it will handle that appropriately.

If you run this example, you should see the same output for both:
```
const dateString = '2024-01-09T00:00:00.000Z';

const parsedDate = Date.parse(dateString);
const constructedDate = new Date(dateString);

console.log(JSON.stringify({ parsed: parsedDate, constructed: constructedDate.getTime() }));
```

Can we also use `const` instead of `var` for `d` here as well?

Reply:
Updated! Thanks for the pointer.